security/v1/types.go

```diff
@@ -135,7 +136,7 @@ var (
 // AllowedFlexVolume represents a single Flexvolume that is allowed to be used.
 type AllowedFlexVolume struct {
 	// Driver is the name of the Flexvolume driver.
-	Driver string `json:"driver,omitempty" protobuf:"bytes,1,opt,name=driver"`
+	Driver string `json:"driver" protobuf:"bytes,1,opt,name=driver"`
```

Comment:
This change reflects upstream type definition: https://github.com/kubernetes/kubernetes/pull/53179#discussion_r150297315